src/core/libmaven/isotopeDetection.cpp

```diff
@@ -205,6 +205,11 @@ bool IsotopeDetection::filterIsotope(Isotope x, float isotopePeakIntensity, floa
     //here w should really be determined by the minRt and maxRt for the parent and child peaks
     if (parentGroup)
     {
+        float isotopeMass = x.mass;
+        Compound* compound = parentGroup->getSlice().compound;
```

Comment:
Suggestion: using `parentGroup->getCompound()` here would give you the same object, and seems more succinct.